src/_includes/partials/token_table.njk

```diff
@@ -0,0 +1,63 @@
+<table class="table-display-token-{{type}}">
```

Comment:
Can you add some formatting to this file? It's not easy to work with in this state.

Reply:
> Can you add some formatting to this file?

I haven't found a way. I I add formatting, tabs, or spaces, several hundreds of <p> tags appear. Have you ever encountered this problem with liquid?


> Some token names appear in the wrong format

I can check out what you did to convert camel case into dashes. I just print out exactly how they came from the token file. 



> Can we build some mobile responsiveness.

Yes, can we do this after merging? This was just a proof of concept for the partial. We can still improve on it.

Reply:
A few updates:

- Used your code in a new filter to fix token names
- Added the contrast column

Reply:
Another update: I found out about what the dash does in `{{-` and `{%-`. Its to remove white space! And it also removes empty `<p>` tags!